code/controllers/subsystems/initialization/culture.dm

```diff
@@ -26,7 +41,66 @@ SUBSYSTEM_DEF(culture)
 				tagged_info[culture.category] = list()
 			var/list/tag_list = tagged_info[culture.category]
 			tag_list[culture.name] = culture
+
+
+	for(var/jobtype in subtypesof(/datum/job))
+		var/datum/job/job = jobtype
+		var/title = initial(job.title)
+		if(title)
+			dreams |= "a [title]"
+			credits_nouns |= uppertext("the [title]")
+
+	var/list/all_content_packages = decls_repository.get_decls_of_subtype(/decl/content_package_manifest)
+	for(var/package in all_content_packages)
+		var/decl/content_package_manifest/manifest = all_content_packages[package]
+		if(LAZYLEN(manifest.worths))
+			for(var/thing in manifest.worths)
+				worths[thing] = max(manifest.worths[thing], worths[thing])
+		if(LAZYLEN(manifest.dreams))
+			dreams |= manifest.dreams
+		if(LAZYLEN(manifest.credits_other))
+			credits_other |= manifest.credits_other
+		if(LAZYLEN(manifest.credits_adventure_names))
+			credits_adventure_names |= manifest.credits_adventure_names
+		if(LAZYLEN(manifest.credits_crew_names))
+			credits_crew_names |= manifest.credits_crew_names
+		if(LAZYLEN(manifest.credits_holidays))
+			credits_holidays |= manifest.credits_holidays
+		if(LAZYLEN(manifest.credits_adjectives))
+			credits_adjectives |= manifest.credits_adjectives
+		if(LAZYLEN(manifest.credits_crew_outcomes))
+			credits_crew_outcomes |= manifest.credits_crew_outcomes
+		if(LAZYLEN(manifest.credits_topics))
+			credits_topics |= manifest.credits_topics
+		if(LAZYLEN(manifest.credits_nouns))
```

Comment:
Are these lists really LAZY or is `LENGTH()` more appropriate?

Reply:
It doesn't really matter considering `LAZYLEN()` is a macro for `length()`.

Reply:
Programmatically it doesn't matter, no, it's more about what expectations one can have about lists that are used along with LAZY-macros.